### ISSUE TITLE
Neither Summary nor Details tells you where the saved image is

### ISSUE THREAD

**Issue by skierpage** [Tommy-Geenexus/exif-eraser#27]
The most important information about the operation of this utility is _What is the path and filename of the stripped image?_  If you only convert one image, it should display that without you having to click anything. Summary doesn't tell you.
- Instead of just "All done! 1 of 1 images modified", Summary could show a thumbnail of each modified file with its path next to it.

Clicking Details doesn't tell you the path of the stripped image, instead it shows a strange UI:
![Screenshot_20210709-162455](https://user-images.githubusercontent.com/1303668/125144317-53495180-e0d2-11eb-89c5-81bf5bfb5ea0.png)

- Clicking the green (xv Modified) button shows a dialog with the filename (but not its path) of the original file, which is less useful than the filename of ExifEraser's modified version, the extension (why not show this with the filename?), and green checked buttons for (✓ICC Profile) and (✓EXIF). But the latter two buttons don't do anything even though they look exactly like the (Modified) button.
  - On my Pixel 3 with large font in portrait orientation, the (EXIF) button is slightly clipped (see screenshot). It turns out there's more info in this dialog visible in landscape orientation, a red (✕ Photos) button. I have no idea what this means, I'm saving in the DCIM/Camera folder... I think. Nope, it turns out I can scroll this list of buttons and it's really (✕ Photoshop), followed by (✓XMP), (✓ExtendedXMP). _There should be some indication that this row of not-buttons is scrollable! Or, since they're not buttons, maybe have a scrollable list with headings for what these are, e.g. "**Metadata removed**: ICC Profile, EXIF, XMP, ExtendedXMP ; **Metadata not removed**: Photoshop"
- Going back to the Details page, I hoped that the green (✓Saved) button will tell me the name of the saved file, but it just flashes the whole image card.
- It's weird having the same green rounded button do/not do three different things.

My workaround is to go into the Android Files application and browse Images > Sort by > Newest date first; ExifEraser's copy is at or near the top, with ` (1)` appended to the filename. This is workable, but very indirect. My workflow is usually use Google Photos to crop and correct the image, then ExifEraser, then I usually wind up deleting the original(s) and renaming the stripped file. If ExifEraser could present the filename of the stripped file in Summary or Details then maybe there's a way clicking on it could prompt to launch an app, such as Photos or Files.

**Comment by Tommy-Geenexus** [comment]
> The most important information about the operation of this utility is _What is the path and filename of the stripped image?_ If you only convert one image, it should display that without you having to click anything. Summary doesn't tell you.
> 
You as the user explicitly select the path the image will be saved to. Also the file name will always be the original file name + optional suffix (if set in the app settings).

>Clicking Details doesn't tell you the path of the stripped image, instead it shows a strange UI:
>
The details are intended to only display the most important stuff, i.e. whether the image has been modified and saved.

>Clicking the green (xv Modified) button shows a dialog with the filename (but not its path) of the original file, which is less useful than the filename of ExifEraser's modified version, the extension (why not show this with the filename?), and green checked buttons for (✓ICC Profile) and (✓EXIF). But the latter two buttons don't do anything even though they look exactly like the (Modified) button.
On my Pixel 3 with large font in portrait orientation, the (EXIF) button is slightly clipped (see screenshot). It turns out there's more info in this dialog visible in landscape orientation, a red (✕ Photos) button. I have no idea what this means, I'm saving in the DCIM/Camera folder... I think. Nope, it turns out I can scroll this list of buttons and it's really (✕ Photoshop), followed by (✓XMP), (✓ExtendedXMP). _There should be some indication that this row of not-buttons is scrollable! Or, since they're not buttons, maybe have a scrollable list with headings for what these are, e.g. "Metadata removed: ICC Profile, EXIF, XMP, ExtendedXMP ; Metadata not removed: Photoshop"
>
The name shown is that of the modified file, if possible. You are likely saving the file under the source directory with the same name (no suffix), in which case the file will be renamed before saving, which the app can't detect.

Green means that the image contained that type of metadata and it was removed, red simply means the image did not contain that type of metadata.
It is a standard implementation as in any app using chips in a group, and as with other apps a clipped end signals that there is more content to reveal.

>Going back to the Details page, I hoped that the green (✓Saved) button will tell me the name of the saved file, but it just flashes the whole image card.
>
This is something that i can see being useful, showing a dialog which explicitly contains the save path.

>My workaround is to go into the Android Files application and browse Images > Sort by > Newest date first; ExifEraser's copy is at or near the top, with (1) appended to the filename. This is workable, but very indirect. My workflow is usually use Google Photos to crop and correct the image, then ExifEraser, then I usually wind up deleting the original(s) and renaming the stripped file. If ExifEraser could present the filename of the stripped file in Summary or Details then maybe there's a way clicking on it could prompt to launch an app, such as Photos or Files.
>
You have the ability to view the full path of any image processed, simply by clicking on the image in the details view and opening it in your gallery of choice. From there you may view the image details, including the save path.
You can also add a suffix to any stripped image via settings -> display name suffix.


TL;DR i will open a new issue regarding showing the save path by clickling the "saved" chip and consider implementing it in a future version

**Comment by Tommy-Geenexus** [comment]
@skierpage 
The issue regarding showing an incorrect name under details -> modified has been fixed.
Also selecting details -> saved will now show a dialog with the canonical save path of the image on Android 8+
